torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -396,6 +396,16 @@ bool GpuLower::hasCurrent() {
 
 void GpuLower::propagateExprInfo(const Expr* old_expr, const Expr* new_expr) {
   pred_elimination_.propagateRemovalInfo(old_expr, new_expr);
+  if (old_expr->isA<kir::Allocate>()) {
```

Comment:
`kir::Allocate` can now be replaced, so the allocation info map needs to be updated. Not an ideal way. Needs more robust way to keep these states up-to-date.